codacy/values-production.yaml

```diff
@@ -66,23 +66,28 @@ codacy-ingress:
   create: true
   ingress:
     hostname:
-      app: <--- codacy-app.dns.internal ---> # Codacy application DNS 
+      app: <--- codacy-app.dns.internal ---> # Codacy application DNS
       api: <--- codacy-api.dns.internal ---> # Codacy API DNS
     annotations:
       kubernetes.io/ingress.class: nginx
       kubernetes.io/tls-acme: "true"
       certmanager.k8s.io/acme-challenge-type: http01
```

Comment:
If you are changing this lets remove any certmanager and external dns reference from this ingress. 

Also any "tls thing"